web/src/app/forms/FormContainer.js

```diff
@@ -117,50 +77,71 @@ export class FormContainer extends React.PureComponent {
         return i !== parseInt(idx, 10)
       })
 
-      return this.props.onChange(
+      return props.onChange(
         mapOnChangeValue(set(mapValue(oldValue), arrayPath, newArr)),
       )
     }
 
-    return this.props.onChange(
+    return props.onChange(
       mapOnChangeValue(set(mapValue(oldValue), fieldName, value)),
     )
   }
 
-  render() {
-    return <FormContext.Consumer>{this.renderComponent}</FormContext.Consumer>
```

Comment:
Thank you so much for reviewing my PR. Is it okay to just return `  return <FormContext.Consumer>{renderComponent}</FormContext.Consumer>` since we want to convert this code into hook?